x-pack/qa/openldap-tests/build.gradle

```diff
@@ -1,18 +1,16 @@
 apply plugin: 'elasticsearch.standalone-test'
```

Comment:
its not `elasticsearch.standalone-rest-test` but `elasticsearch.standalone-test`. IMO we should keep it. The others are rest tests. this one isn't

Reply:
Got it. The question remains, what's the point of this plugin? Why don't we just use the `elasticsearch.java` plugin for these projects?

Reply:
the main thing this does is that we just have one sourceSet instead of the default two. I doubt that gives us very much in terms of performance these days tbh

Reply:
Right, the `main` source set tasks should end up being skipped due to `NO-SOURCE`. I think that's fine and much better than maintaining a separate plugin that likely uses slightly different conventions.